radixdlt-cli/src/main/groovy/com/radixdlt/cli/ShowValidatorConfig.groovy

```diff
@@ -54,15 +55,15 @@ class ShowValidatorConfig implements Runnable {
 		def latestValidatorRegistration = api.getAtomStore().getUpParticles(api.getAddress(), null)
 				.filter({ particle -> particle instanceof RegisteredValidatorParticle })
 				.findFirst()
-		if (latestValidatorRegistration.isPresent()) {
+
+		latestValidatorRegistration.ifPresentOrElse(validator -> {
 			println "Current validator configuration at ${address}:"
-			def validator = latestValidatorRegistration.get()
 			def url = validator.getUrl()
 			def allowedDelegators = validator.getAllowedDelegators()
 			printf("  url: %s%n", url == null ? "<not set>" : url)
 			printf("  allowedDelegators: %s%n", allowedDelegators == null ? "<not set, allows any>" : allowedDelegators)
-		} else {
-			println "No active validator configuration at ${address}"
-		}
+		},
+			() -> println "No active validator configuration at ${address}")

```

Comment:
Would prefer to see closing paren for the ifPresentOrElse on a newline with the same indentation as the previous closing brace.